dictionaries/software-terms/src/software-tools.txt

```diff
@@ -5,6 +5,7 @@ airbrake
 AirWatch
 AKS
 Alibaba
+all-contributorsrc
```

Comment:
Adding an inline comment could help as this one is not obvious

Reply:
Done!